### PR TITLE
updated changelog 0.25.1 --> issues 43 and 44 of larray-editor project.

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: doc/source/changes/version_0_25_1.rst.inc [hunk 1]
```diff
@@ -19,6 +19,11 @@ Fixes
 
 * fixed displaying empty array when starting the viewer or a new session in it.
 
+* fixed default import of `view`, `edit` and `compare` functions in the viewer.
+
+* fixed rezising to content of rows and columns in the viewer
```
Comment (gdementen) [inline doc/source/changes/version_0_25_1.rst.inc]: typo in resizing

Comment (gdementen) [reply doc/source/changes/version_0_25_1.rst.inc]: why not a simpler sentence, eg:
* fixed row and column resizing by double clicking on the edge of an header cell.

File: doc/source/changes/version_0_25_1.rst.inc [hunk 2]
```diff
@@ -19,6 +19,11 @@ Fixes
 
 * fixed displaying empty array when starting the viewer or a new session in it.
 
+* fixed default import of `view`, `edit` and `compare` functions in the viewer.
```
Comment (gdementen) [inline doc/source/changes/version_0_25_1.rst.inc]: Our users wouldn't understand that I think. I would write:

    fixed the `view()`, `edit()` and `compare()` functions not being available in the viewer console.

At least the word console needs to be in there

**Comment by alixdamman** [comment]
updated

**Pull request merged**